xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -20,8 +20,45 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>An assembly load context represents the runtime's concept of a scope for binding. This class is abstract.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> represents a load context. Conceptually, a load context creates a scope for loading, resolving, and potentially unloading a set of assemblies.
+ 
+ <xref:System.Runtime.Loader.AssemblyLoadContext> is an abstract class. The <xref:System.Runtime.Loader.AssemblyLoadContext.Load(System.Reflection.AssemblyName)?displayProperty=nameWithType> needs to be implemented to create a concrete class.
+ 
+ The <xref:System.Runtime.Loader.AssemblyLoadContext> exist primarily exists to provide assembly isolation. It allows multiple versions of the same assembly to be loaded within a single process. It replace the isolation mechanisms provided by multiple <xref:System.AppDomain> instances in the .NET Framework.
```

Comment:
Nit: Probably want to leave out one of the "exist"

Reply:
Nit: I would write it as "provide assembly **binding** isolation" (I personally would think that "assembly isolation" means isolation of everything including instances and so on).

Reply:
"It replace**s**"